### ISSUE TITLE
add aria-label to describe empty link purpose at scrollable horizontal menu with a toggle control

### ISSUE THREAD

**Issue by snaphuman** [pure-css/pure-site#401]
### Description

Running a web accessibility evaluation tool reports that anchor element with id=toggle it's empty and does not provide it's purpose in context according to the success criteria 2.4 of wcag 2.1.

### Expected behavior

The anchor element should have a text that indicates the function or purpose of the link otherwise it can introduce confusion for keyboard and screen reader users.

### Actual behavior

Adding aria-label attribute to the element keeps the default behavior for the toggle control and ensures that accessibility test passes;

`<a href="#" class="custom-menu-toggle" id="toggle" aria-label="click to expand menu"><s class="bar"></s><s class="bar"></s></a>`

### Steps to Reproduce
1. Navigate to [Responsive Horizontal-to-Scrollable Menu](https://purecss.io/layouts/tucked-menu/) 
2. Validate the page with a accessibility tool. I used [wave](https://addons.mozilla.org/en-US/firefox/addon/wave-accessibility-tool/)
3. Failed Test [result](https://imagebin.ca/v/4Z6i6XZFNUFz)

### Environment
* Browser name and version: Firefox developer edition 66.0b1
